keylime/src/tpm.rs

```diff
@@ -613,6 +624,99 @@ impl Context {
         Ok(ak_handle)
     }
 
+    /// Load a key handle from a string of the handle location
+    /// If a password is supplied, authorise the handle
+    /// # Arguments
+    ///
+    /// `handle` : The string of the handle, eg. from config
+    /// `password` ; The string password, to be converted to hex if there is the "hex:" prefix
+    ///
+    /// # Return
+    /// The corresponding KeyHandle, or a TPMError
+    pub fn get_key_handle(
```

Comment:
I see you declared this method as public, but it is currently only used to get the handles for IDevID and IAK keys. 

Do you see future usage of this in other use case? If not, could you make it private by removing the `pub` keyword?

Reply:
I thought it could be a useful method but did not have a specific use-case in mind. I'll change to private and we can always change back to public if we want to use it in future.